lib/net/pool.js

```diff
@@ -704,6 +729,60 @@ class Pool extends EventEmitter {
     this.compactBlocks.clear();
   }
 
+  /**
+   * Start the filters headers sync.
+   */
+
+  async startFilterHeadersSync() {
+    this.filterSyncing = true;
+    this.logger.info('Starting filter headers sync (%s).',
+      this.chain.options.network);
+    if (!this.opened || !this.connected)
+      return;
+
+    const cFHeaderHeight = this.cfHeaderChain.tail.height;
+    const startHeight = cFHeaderHeight
+    ? cFHeaderHeight + 1 : 1;
+    const chainHeight = await this.chain.height;
+    const stopHeight = chainHeight - startHeight + 1 > 2000
+     ? startHeight + 1999 : chainHeight;
```

Comment:
this should do the same
```suggestion
const stopHeight = Math.min(chainHeight, startHeight + 1999);
```